light-zk.js/src/transaction/transaction.ts

```diff
@@ -576,10 +577,7 @@ export class Transaction {
     var inputMerklePathElements = new Array<Array<string>>();
     // getting merkle proofs
     for (const inputUtxo of inputUtxos) {
-      if (
-        inputUtxo.amounts[0] > new BN(0) ||
-        inputUtxo.amounts[1] > new BN(0)
-      ) {
+      if (inputUtxo.amounts[0] > BN_0 || inputUtxo.amounts[1] > BN_0) {
```

Comment:
I am not sure that this will work
```suggestion
      if (inputUtxo.amounts[0].gt(BN_0) || inputUtxo.amounts[1].gt(BN_0)) {
```